pages/pie-chart/with-links.page.tsx

```diff
@@ -0,0 +1,106 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useContext } from 'react';
+
+import SpaceBetween from '~components/space-between';
+import Link from '~components/link';
+import PieChart from '~components/pie-chart';
+import ScreenshotArea from '../utils/screenshot-area';
+import AppContext, { AppContextType } from '../app/app-context';
+import FocusTarget from '../common/focus-target';
+
+type DemoContext = React.Context<
+  AppContextType<{
+    useLinks: 'keys' | 'values' | null;
+  }>
+>;
+
+import { commonProps, data1, FoodData, segmentDescription1 } from './common';
+
+export default function () {
+  const { urlParams, setUrlParams } = useContext(AppContext as DemoContext);
+
+  return (
+    <>
+      <h1>Pie chart with links</h1>
```

Comment:
Can we make it simpler by rendering two separate charts, one with value links and one with key links? This would allow to remove extra elements and will make it easier to understand the setup.

Reply:
Done by splitting it in two pages. This will make it more convenient for eventual visual regression tests.